src/app/components/progress-indicator/waterfall-progress-indicator/index.html

```diff
@@ -0,0 +1,325 @@
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-separate">
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-6.png" />
+              <div class="caption caption-danger sky-text-danger">Don't present waterfall progress indicators before the extent of tasks is known.</div>
+            </div>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-7.png" />
+              <div class="caption caption-success sky-text-success">Do present decisions that determine the number and order steps before showing waterfall progress indicators.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+          <sky-docs-visual-example-text>
+            <p>Don’t use waterfall progress indicators to edit properties after completing configuration processes. Use <a stacheRouterLink="/components/modal">modals</a> instead.</p>
+            <p>If configuration properties are interrelated and require users to reexamine all or most properties, then reframe the editing task as “starting over” and restart the waterfall process.</p>
+          </sky-docs-visual-example-text>
+          <sky-docs-visual-example-image>
+            <div class="sky-margin-stacked-separate">
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-8.png" />
+              <div class="caption caption-danger sky-text-danger">Don't revert finished states to edit properties in waterfall progress indicators.</div>
+            </div>
+            <div>
+              <img src="~/assets/img/guidelines/progress-indicator/waterfall-progress-usage-9.png" />
+              <div class="caption caption-success sky-text-success">Do group related properties and let users edit them in isolation after they complete waterfall progress indicators.</div>
+            </div>
+          </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+      </sky-demo-page-content>
+    
+      <sky-demo-page-content sectionHeading="Anatomy">
+        <sky-docs-anatomy imagePath="~/assets/img/guidelines/progress-indicator/waterfall-progress-anatomy.png">
+          <sky-docs-anatomy-item anatomyNumber="1">Step indicator</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="2">Step label</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="3">Completed step action</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="4">Active step action</sky-docs-anatomy-item>
+          <sky-docs-anatomy-item anatomyNumber="5">Waterfall title</sky-docs-anatomy-item>
```

Comment:
Why is the waterfall title #5? Given that it’s at the top of the page, it seems like it should be #1. But given that it’s included in the Options section, I assume that the problem is that this should actually be marked as optional. Is that correct?

Reply:
Ported over to #625.